### PR TITLE
Add Section Component and Relationships

### PR DESCRIPTION
**Pull request by aabidsofi19**
**Notes for Reviewers**

This PR fixes #
- Added the Section Component 
- Added a  hierarchial relation from section to  every component (*)
-  updated the styleoverrides for components 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by aabidsofi19** [comment]
@leecalcote  i have updated the spreadsheet to use native functions , and the behavior is more like  previous ( i.e we are inserting empty polygon-points , data , etc )

File: server/meshmodel/accurate/0.5.0/SubNamespace.json [hunk 1]
```diff
@@ -1 +1 @@
...
...Accurate","primaryColor":"#14191f","published":false,"secondaryColor":"#14191f","shape":"circle","styleOverrides":"","subCategory":"Security \u0026 Compliance","svgColor":"","svgComplete":"","svgWhite":""},"model":{"name":"accurate","version":"0.5.0","displayName":"Accurate","category":{"name":"Provisioning","metadata":null},"metadata":{"source_uri":"https://github.com/cybozu-go/accurate/releases/download/accurate-chart-0.7.0/accurate-0.7.0.tgz"}},"schema":"{\n \"description\": \"SubNamespace is the Schema for the subnamespaces API\",\n \"properties\": {\n  \"spec\": {\n   \"description\": \"Spec is the spec of SubNamespace.\",\n   \"properties\": {\n    \"annotations\": {\n     \"additionalProperties\": {\n      \"type\": \"string\"\n     },\n     \"description\": \"Annotations are the annotations to be propagated to the sub-namespace.\",\n     \"type\": \"object\"\n    },\n    \"labels\": {\n     \"additionalProperties\": {\n      \"type\": \"string\"\n     },\n     \"description\": \"Labels are the labels to be propagated to the sub-namespace\",\n     \"type\": \"object\"\n    }\n   },\n   \"type\": \"object\"\n  }\n },\n \"title\": \"Sub Namespace\",\n \"type\": \"object\"\n}"}
```
Comment (aabidsofi19) [inline server/meshmodel/accurate/0.5.0/SubNamespace.json:1]: now the styleOverrides is empty if properties are not set .

**Review by MUzairS15 — COMMENTED** [review]
Can you tag the relationship?

**Comment by leecalcote** [comment]
@aabidsofi19 "Added a hierarchial relation for section to any component (*)" to any or to every?

**Comment by leecalcote** [comment]
> @aabidsofi19 "Added a hierarchial relation for section to any component (*)" to any or to every?

@aabidsofi19 oh, by "component", you mean "shape". Ok, good. Although, the designator for this is "parent".

**Comment by aabidsofi19** [comment]
> > @aabidsofi19 "Added a hierarchial relation for section to any component (*)" to any or to every?
> 
> @aabidsofi19 oh, by "component", you mean "shape". Ok, good. Although, the designator for this is "parent".

from  section to every component ( not only shapes )  [ i think i am using wrong terminology to describe it ]

File: server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          "model": "*"
+        }
+      ]
+    },
+    "deny": {
+      "from": [],
+      "to": []
+    }
+  }
+}
```
Comment (aabidsofi19) [inline server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json:37]: @leecalcote  @MUzairS15  here is the relationship

Comment (leecalcote) [reply server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json:37]: LGTM... unless relationships are not sourced from any model other than kubernetes. @MUzairS15 is that accurate or do we now support Relationship import for other Models?

**Review by leecalcote — APPROVED** [review]

File: server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json (hunk 2 above)
Comment (MUzairS15) [reply server/meshmodel/kubernetes/relationships/hierarchial_parent_section.json:37]: yes

**Pull request merged**